docker/join.bash

```diff
@@ -1,11 +1,43 @@
 #!/usr/bin/env bash
 #
-# Typical usage: ./join.bash vrx
+# Typical usage: ./join.bash dave_nvidia
```

Comment:
Rocker currently does not have any way to specify the name of the image. It only allows you to give the name of the container, and it gives the image no name (other than the ID hash). If we want to continue to use the image name as the argument to `join.bash` the user will need to either look at the rocker output or use docker command-line tools to look up the correct image hash by checking the ID of the container.

There are a bunch of ways around this, but the easiest thing is probably to specify the container name using rocker and modify the join script to take a container name as an argument directly. Then it doesn't have to try to look up the container name from the image ID.

Reply:
For example, the rocker command would be:
```
rocker --devices /dev/input/js0 --dev-helpers --nvidia --x11 --user --home --name <CONTAINER NAME> --git dave_nvidia:latest
```
And the argument to the join script would be:
```
./join_bash <CONTAINER NAME>
```